tensorflow/python/training/experimental/mixed_precision.py

```diff
@@ -73,39 +73,96 @@ def _wrap_optimizer(opt, loss_scale):
 
 @tf_export(v1=['train.experimental.enable_mixed_precision_graph_rewrite'])
 def enable_mixed_precision_graph_rewrite(opt, loss_scale='dynamic'):
-  """Enable mixed precision via a graph rewrite.
-
-  Mixed precision is the use of both float16 and float32 when training a model,
-  and is used to make the model run faster. This function will use mixed
-  precision to speed up the execution time of your model when run on a GPU. It
-  does this by changing the dtype of certain operations in the graph from
-  float32 to float16.
-
-  This function additionally wraps an Optimizer with a LossScaleOptimizer, which
-  is required to prevent underflow in the float16 tensors during the backwards
-  pass. An optimizer must be passed to this function, which will then be wrapped
-  to use loss scaling.
+  """Enables faster training on compatible hardware accelerators by using a
+  mix of numerical data types. Known as mixed precision, this is achieved
+  via a graph rewrite operation and a scaled optimizer.
+  
+  Mixed precision is the use of both float16 and float32 tensors when training
+  a model. Performing only certain arithmetic operations in lower precision
+  (float16) results in performance gains of when using compatible hardware
+  accelerators. This decreases training time and reduces memory usage,
+  typically without sacrificing model performance.
+
+  Example:
+
+  ```python
+  model = tf.keras.models.Sequential([
+      ...
+  ])
+
+  opt = tf.keras.optimizers.SGD()
+  opt = tf.train.experimental.enable_mixed_precision_graph_rewrite(opt)
+
+  model.compile(loss="categorical_crossentropy",
+                optimizer=opt,
+                metrics=["accuracy"])
+
+  model.fit(x_train, y_train,
+            batch_size=batch_size,
+            epochs=epochs)
+  ```
+
+  For a complete example showing the speed-up on training an image classification
+  task on CIFAR10, check out this
+  [Colab notebook](https://colab.research.google.com/drive/1qabCQKJZhdIDOrXuhoLa9uMB4RJJ2a1N).
```

Comment:
The notebook is here: https://github.com/tensorflow/examples/pull/65
The link should probably be updated to the tensorflow/examples repo once that PR is merged